### ISSUE TITLE
[Alpha v14.0] - Object Reference Error on Obsidian Registration Entry Block

### ISSUE THREAD

**Issue by mikedotmundy** [SparkDevNetwork/Rock#5135]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Did you perform a search at https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ARock to see if your bug or enhancement is already reported?
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?

### Description
Creating just a simple registration and using the Obsidian Registration Entry block causes an error message and the Person is not registered.

Stack Trace:
```
 at Rock.Blocks.Event.RegistrationEntry.SubmitRegistration(RockContext rockContext, RegistrationContext context, RegistrationEntryBlockArgs args, String& errorMessage) in C:\Github\Rock\Rock.Blocks\Event\RegistrationEntry.cs:line 852
   at Rock.Blocks.Event.RegistrationEntry.SubmitRegistration(RegistrationEntryBlockArgs args) in C:\Github\Rock\Rock.Blocks\Event\RegistrationEntry.cs:line 372
```

#### A Picture Is worth a Thousand Words
<img width="1728" alt="Screen Shot 2022-08-23 at 19 20 01" src="https://user-images.githubusercontent.com/23226974/186283306-8f64a529-700f-4a54-befd-47d87abc24af.png">
<img width="1728" alt="Screen Shot 2022-08-23 at 19 20 30" src="https://user-images.githubusercontent.com/23226974/186283323-5a1a9a73-bba0-433b-9cb9-153b65ba501b.png">

### Steps to Reproduce

1. Create a new Registration Template, keeping all the default settings.
2. Create a new Registration Instance, keeping all the default settings (give it a Registration Start Date).
3. Add the Obsidian Registration Entry block to the Registration page, and try to register for the event. I used a url as follows: `https://prealpha.rocksolidchurchdemo.com/registration?registrationinstanceid=11`
4. See error message displayed.

**Expected behavior:**

No error message

**Actual behavior:**

There is an error message and you are unable to register.

### Versions

* **Rock Version:** [v14.0 (may also be an issue in earlier versions - unable to test)]
* **Client Culture Setting:** [en-US]

**Comment by JimMichael** [comment]
We can confirm we're seeing this exception on Obsidian registrations as well.

**Comment by cabal95** [comment]
This should be fixed by d4eb11c4c0399980324ee7fe8696589ebd3ba60f
